content/en/blog/2022/prow-and-tide-for-kubernetes-contributors/prow-and-tide-for-kubernetes-contributors.md

```diff
@@ -5,26 +5,65 @@ date: 2022-05-31
 slug: prow-and-tide-for-kubernetes-contributors
 ---
 
-**Authors:** [Chris Short](https://github.com/chris-short)
+**Authors:** [Chris Short](https://github.com/chris-short) (edited by [Frederico Muñoz] (https://github.com/fsmunoz))
```

Comment:
I suggest just doing Authors: Chris Short, Frederico Munoz.  At this point, you've written at least as much text as Chris did.